GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -330,21 +352,77 @@ - (instancetype)init {
   return self;
 }
 
+/** Populates a unique NSUInteger as GDTCORBackgroundIdentifier for given semaphore and maps it to
+ * the semaphore in NSMutableDictionary.
+ *
+ * @param semaphore The task semaphore needs to be mapping.
+ * @return A unique GDTCORBackgroundIdentifier mapped to the given semaphore.
+ */
++ (GDTCORBackgroundIdentifier)populateBgIDwithSemaphore:(dispatch_semaphore_t)semaphore {
```

Comment:
maybe rename to createAndMapBackgroundIdentifierToSemaphore

Reply:
Renamed.